packages/react-instantsearch/src/core/createConnector.js

```diff
@@ -143,9 +144,10 @@ export default function createConnector(connectorDesc) {
         error,
         widgets,
         metadata,
+        resultsFacetValues,
       } = store.getState();
       const searchState = {results, searching, error};
-      return connectorDesc.getProvidedProps.call(this, props, widgets, searchState, metadata);
+      return connectorDesc.getProvidedProps.call(this, props, widgets, searchState, metadata, resultsFacetValues);
```

Comment:
Ideally, it could be nice to be able to get search results for facet values from the searchResults object of the helper. Is that feasible? cc @bobylito